deepmd/data_generation/convert.py

```diff
@@ -0,0 +1,22 @@
+import dpdata
+import numpy as np
+
+data = dpdata.LabeledSystem("h2o_32", fmt="qe/cp/traj")
```

Comment:
Could we add links to the documentation that explain how the conversion should be done as a comment?